Example/Messaging/Tests/FIRMessagingServiceTest.m

```diff
@@ -146,12 +146,15 @@ - (void)testSubscribeWithInvalidTopic {
   XCTestExpectation *exceptionExpectation =
   [self expectationWithDescription:@"Should throw exception for invalid token"];
   @try {
+#pragma clang diagnostic push
+#pragma clang diagnostic ignored "-Wnonnull"
```

Comment:
nit: When testing, you can just cast `nil` pointers to nonnull types. `(id _Nonnull)nil`